R/get_awards.R

```diff
@@ -0,0 +1,101 @@
+#' Get NSF Arctic/Polar program award information
+#'
+#' Uses the [NSF API](https://www.research.gov/common/webapi/awardapisearch-v1.htm)
+#' to get all records pertaining to the Arctic or Polar programs.
+#'
+#' @param from_date (character) Optional. Returns all
```

Comment:
Up to your style preference but you can go over the 80 character limit for `@param` definitions